terraform/module-ses/redis.tf

```diff
@@ -1,50 +1,51 @@
 resource "aws_security_group" "redis" {
-  count       = "${var.create_elasticache == "true" ? 1 : 0}"
+  count       = var.create_elasticache == "true" ? 1 : 0
   name        = "${var.project}-${var.elasticache_name}-${var.env}"
   description = "${var.elasticache_name} ${var.env} for ${var.project}"
-  vpc_id      = "${var.vpc_id}"
+  vpc_id      = var.vpc_id
 
   ingress {
-    from_port = "${var.elasticache_port}"
-    to_port   = "${var.elasticache_port}"
+    from_port = var.elasticache_port
+    to_port   = var.elasticache_port
     protocol  = "tcp"
 
-    security_groups = ["${var.elasticache_security_groups}"]
+    security_groups = var.elasticache_security_groups
```

Comment:
```suggestion
    security_groups = [var.elasticache_security_groups]
```

Reply:
`elasticache_security_groups` is already an array. https://console.cycloid.io/projects/cycloid-owl/ci-stack-ses/pipelines/ci-stack-ses-master/jobs/terraform-test-pr/builds/114200#tests. We can not use an array of array for `security_groups`.